src/lib.rs

```diff
@@ -38,72 +34,136 @@ impl rowan::Language for NixLanguage {
 pub type SyntaxNode = rowan::SyntaxNode<NixLanguage>;
 pub type SyntaxToken = rowan::SyntaxToken<NixLanguage>;
 pub type SyntaxElement = rowan::NodeOrToken<SyntaxNode, SyntaxToken>;
+pub type SyntaxElementChildren = rowan::SyntaxElementChildren<NixLanguage>;
+pub type SyntaxNodeChildren = rowan::SyntaxNodeChildren<NixLanguage>;
 
-/// A convenience function for first tokenizing and then parsing given input
-pub fn parse(input: &str) -> AST {
-    parser::parse(Tokenizer::new(input))
-}
```

Comment:
I think this should say, even if it's just an alias for `Root::parse`. It provides a more obvious "entry point" to the crate.

Reply:
We can just document that `Root::parse` is the entry point.